_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,213 @@
+---
+id: b8bb68
+name: Text spacing set in the `style` attribute is below permissible minimum value or not set to `!important`
+rule_type: atomic
+description: |
+  This rule checks that
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] and has any of the below CSS properties set in the `style` attribute:
+
+- [word-spacing][]
+- [letter-spacing][]
+- [line-height][]
+
+## Expectation 1
+
+The test target does not have "important" as its [word-spacing][] [priority][], unless the [computed][] [word-spacing][] is `0.16em` or greater.
+
+## Expectation 2
+
+The test target does not have "important" as its [letter-spacing][] [priority][], unless the [computed][] [letter-spacing][] is `0.12em` or greater.
+
+## Expectation 3
+
+The test target does not have "important" as its [line-height][] [priority][], unless the [computed][] [line-height][] is `1.5em` or greater.
```

Comment:
All three of these attributes compute to absolute lengths, and hence pixel values, so it doesn't make sense to talk about ems. I'm thinking something like this could make sense instead:

> [...] unless the computed `<property>` is at least `<number>` times the computed `font-size`.

Reply:
The required space is relative to the font size in the SC. EM seems like the right unit for that. How would you propose to do it if not through em? I guess you can express it as a percentage of the font-size, but that's effectively what EM does.

Reply:
We cannot talk about ems when dealing with pixels, exactly because ems are a relative measure and pixels aren't. It's comparing apples and potatoes. See my suggestion above though ☝️